samples/aspnetcore/blazor/FlightFinder/FlightFinder.Client/NavMenu.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@using Microsoft.AspNetCore.Blazor.Routing
+<nav>
+    <ul class="nav">
+        <li class="nav-item">
+            <NavLink ActiveClass="active" href="/" Match="NavLinkMatch.All" class="nav-link">Search Flights</NavLink>
+        </li>
+        <li class="nav-item">
+            <NavLink ActiveClass="active" href="/cars" Match="NavLinkMatch.All" class="nav-link">Search Car Rentals</NavLink>
+        </li>
```

Comment:
We could consider implementing tabs in a future update. For right now I just did the straightforward thing.

Reply:
What you have here already looks perfectly good as a simple tabs implementation, for tabs that manage their state in terms of URL navigations. No reason why it should be any more complex that I can think of.